docs/platforms/javascript/common/install/late-initializtion.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: Late Initialization (ESM or CJS)
+sidebar_order: 12
+description: "Learn about running Sentry in an ESM or CJS application, in scenarios where you cannot run init early."
+supported:
+  - javascript.node
+  - javascript.connect
+  - javascript.express
+  - javascript.fastify
+  - javascript.hapi
+  - javascript.koa
+---
+
+<Alert>
+  Are you unsure if you should use this installation method? Review our
+  [installation methods](../).
+</Alert>
+
+In order for auto-instrumentation to work, it is generally required to run `Sentry.init()` as early as possible, before anything else is imported in your application.
+
+However, in some cases this may not be possible to do - for example, if you are fetching your DSN from an external source. In this case, you can use the `@sentry/node/preload` hook to ensure modules are wrapped early, which allows you to call `Sentry.init()` later at a time of your choosing.
```

Comment:
```suggestion
However, in some cases this may not be possible to do—for example, if you are fetching your DSN from an external source. In this case, you can use the `@sentry/node/preload` hook to ensure modules are wrapped early, which allows you to call `Sentry.init()` later at a time of your choosing.
```

Reply:
I guess this is an em-dash, which is nice, but should we not still have spaces around this? Seems (visually) a bit weird to me without the spaces...?

Reply:
No space around emdashes. I was taught my Sigrid :)
https://www.thepunctuationguide.com/em-dash.html#spaces

Reply:
I don't like it :D But I accept it ;) 

It does say there though:

> Spaces with the em dash
The em dash is typically used without spaces on either side, and that is the style used in this guide. **Most newspapers, however, set the em dash off with a single space on each side.**

but 🤷 no strong feelings ;)

Reply:
tbh I've also always added spaces. If the em-dash is visibly large enough then I don't mind without spaces but sometimes they are very similar (e.g. here on github).

fwiw on the actual docs page it loo
<img width="224" alt="Screenshot 2024-05-28 at 10 56 05" src="https://github.com/getsentry/sentry-docs/assets/168741329/38e8d8ce-c380-435e-adad-a89591fd45a7">
ks a bit better:

Reply:
I think we should pick one style and go with it (currently we have it without spaces).